Helper/Sale.php

```diff
@@ -74,11 +74,18 @@ function getSales()
 
             $productID = $item->getData('product_id');
             $quantity = (string)($item->getData('qty_ordered') - ($item->getData('qty_canceled') + $item->getData('qty_refunded')));
-
+            
+            // This has returned "" in the wild
             $storeId = $item->getData('store_id');
-            $zones = $this->getTimezone(array($storeId));
-            $zone = $zones[$storeId];
-            $dt = new \DateTime($item->getData('created_at'), new \DateTimeZone($zone));
+
+            // Normal storeIds start at "1", but the sneaky admin store is "0".
+            if($storeId == "0" || !empty($storeId)){
```

Comment:
do you need both checks? Seems like !empty() covers all cases on its own

Reply:
`empty("0")` returns true in php 😕

`"0"` is a valid store_id  so we had to add the `$storeId == "0"` to check for it

Reply:
+1 to what James said.

PHP considers the string `"0"` a falsey value. The docs say `empty() is essentially the concise equivalent to !isset($var) || $var == false.` Don't love it, but PHP is a little too flexible here.

Reply:
Yeah, makes sense. I remember hating PHP, I just don't always remember all the reasons.